tests/test_async_client.py

```diff
@@ -26,11 +28,24 @@ async def test_publish_subscribe(qos: QoS) -> None:
             assert packets[1].payload == b"\x00\xff\x00\x1f"
 
 
+if sys.version_info < (3, 11):
+
+    class BaseExceptionGroup(BaseException):
```

Comment:
I don't get this. How would you be able to catch an exception you just arbitrarily defined here?

Reply:
I wouldn't. This is a workaround. Python version <3.11 doesn't even *have* exception groups, hence they're not going to be raised in the first place.

Reply:
Uh, yeah, they are, because AnyIO and Trio both use my [backport](https://pypi.org/project/exceptiongroup/) for that.

Reply:
Meh. I missed that. Replaced.